katas/6kyu/counting-duplicates.js

```diff
@@ -0,0 +1,17 @@
+function duplicateCount(text) {
+    const letters = text.toLowerCase().split('');
+    let counter = 0;
+    let memo = {};
+
+    if (!letters.length) return 0;
+
+    letters.forEach(letter => {
+        if (memo[letter] && memo[letter] === 1) {
+            ++counter;
+            memo[letter] = 2;
+        }
+        if (!memo[letter]) memo[letter] = 1;
```

Comment:
I guess, it'd be better to use `else if` here to save 1 line and make the code style better